nemo/collections/nlp/models/language_modeling/megatron/t5_model.py

```diff
@@ -128,8 +190,12 @@ def forward(
             encoder_decoder_attn_mask,
             tokentype_ids=tokentype_ids,
             enc_hidden_states=enc_hidden_states,
+            output_enc_hidden=output_enc_hidden,
```

Comment:
I'd rather this be more descriptive like `output_enc_hidden_only`

Reply:
Changed the variable name.